### DISCUSSION TITLE
iphone16 ios18.2 quantumult x 定位模块在飞行模式和非飞行模式下不一致

### DISCUSSION THREAD

**Discussion by ikevinclub** [NSRingo/.github#29] [Q&A 问答] [answered]
当关闭飞行模式，定位模块正常，quantumult请求活动里显示相关请求地址为gspe21-ssl.ls.apple.com
当打开飞行模式后，定位模块就失效了，地图为高德地图，quantumult请求活动里显示相关请求地址为gspe12-cn-ssl.ls.apple.com

**Comment by VirgilClyne** [comment, marked as answer]
https://github.com/orgs/NSRingo/discussions/25
